<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BallooningDialog</class>
 <widget class="QDialog" name="BallooningDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Memory</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelRubric">
     <property name="text">
      <string>Specify memory allocation for this virtual machine. Dynamic Memory Control allows you to specify a minimum and maximum memory value, from which the XenServer host automatically allocates a suitable memory value based on current demand.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="memoryGroup">
     <property name="title">
      <string>Memory Settings</string>
     </property>
     <layout class="QVBoxLayout" name="memoryGroupLayout">
      <item>
       <widget class="QRadioButton" name="radioFixed">
        <property name="text">
         <string>Fixed allocation</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="fixedLayout">
        <item>
         <spacer name="horizontalSpacer1">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labelFixed">
          <property name="text">
           <string>Memory:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="spinnerFixed">
          <property name="suffix">
           <string> MB</string>
          </property>
          <property name="minimum">
           <double>128.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>128.000000000000000</double>
          </property>
          <property name="value">
           <double>1024.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="radioDynamic">
        <property name="text">
         <string>Dynamic allocation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="dynamicDetailsWidget" native="true">
        <layout class="QVBoxLayout" name="dynamicDetailsLayout">
         <property name="leftMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="VMShinyBar" name="vmShinyBar" native="true">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>80</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="dynamicWidget" native="true">
           <layout class="QFormLayout" name="dynamicFormLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="labelDynMin">
              <property name="text">
               <string>Minimum:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="spinnerDynMin">
              <property name="suffix">
               <string> MB</string>
              </property>
              <property name="minimum">
               <double>128.000000000000000</double>
              </property>
              <property name="maximum">
               <double>999999.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>128.000000000000000</double>
              </property>
              <property name="value">
               <double>512.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelDynMax">
              <property name="text">
               <string>Maximum:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="spinnerDynMax">
              <property name="suffix">
               <string> MB</string>
              </property>
              <property name="minimum">
               <double>128.000000000000000</double>
              </property>
              <property name="maximum">
               <double>999999.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>128.000000000000000</double>
              </property>
              <property name="value">
               <double>2048.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelDMCUnavailable">
        <property name="text">
         <string/>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxDeferReboot">
        <property name="text">
         <string>Apply changes on next reboot (do not restart now)</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VMShinyBar</class>
   <extends>QWidget</extends>
   <header>widgets/vmshinybar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BallooningDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BallooningDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
